lib/encrypt.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/enigma'
+
+enigma = Enigma.new
+
+handle = File.open(ARGV[0], "r")
+
+message = handle.read
+
+handle.close
+
+encrypted_info = enigma.encrypt(message)
```

Comment:
The encrypt runner doesn't take arguments, so it can only pass a message into `enigma.encrypt`. That means the key defaults to a randomly generated key and the date defaults to the present date.